lib/ci-helper.ts

```diff
@@ -834,11 +845,10 @@ export class CIHelper {
         return await mailArchiveGit.processMails(prFilter);
     }
 
-    private async getPRInfo(prNumber: number, pullRequestURL: string):
+    private async getPRInfo(_prNumber: number, pullRequestURL: string):
```

Comment:
Why not remove the `prNumber` parameter altogether?

Reply:
I was trying to minimize the changes but yes, that makes more sense.